src/MultiplayerMod/Multiplayer/Objects/ChoreObjects.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+
+namespace MultiplayerMod.Multiplayer.Objects;
+
+public static class ChoreObjects {
```

Comment:
Maybe I'll make a single "repository" for multiplayer objects, I have a thought at least.

Reply:
Also had the same though :)
Feel free to do it, I'll leave it as is to keep scope of the PR small.